OpenAI_API/Chat/ChatResult.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenAI_API.Chat
+{
+	/// <summary>
+	/// Represents a result from calling the Chat API
+	/// </summary>
+	public class ChatResult : ApiResultBase
+    {
+        /// <summary>
+        /// The identifier of the result, which may be used during troubleshooting
+        /// </summary>
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// The list of choices that the user was presented with during the chat interaction
+        /// </summary>
+        [JsonProperty("choices")]
+        public ChatChoice[] Choices { get; set; }
```

Comment:
This should probably be an `IReadOnlyList<ChatChoice>` instead of an array (which most serializers have no issue with). An array implies the results are mutable and is should usually be [avoided in public properties](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections).